### DISCUSSION TITLE
Block drop chance from explosions

### DISCUSSION THREAD

**Discussion by CalaMariGold** [ACGaming/UniversalTweaks#520] [Tweak Suggestions]
An option in the config to change the chance a block will drop from any explosion. This ideally should have a global chance as well as per block which overrides the global chance.

**Comment by jchung01** [comment]
The global chance is already implemented in the Tweaks config under `Explosion Block Drop Chance`, no?
> as well as per block which overrides the global chance.

I would assume this part is already possible by using the Dropt mod. See [`IRuleMatchHarvester`](https://dropt.readthedocs.io/en/latest/json/syntax/#irulematchharvester)'s `type: EXPLOSION`.
